extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -243,9 +243,9 @@ class QueueProcessor extends EventEmitter {
         }
         if (destConfig.transport === 'https') {
             this.destHTTPAgent = new HttpsAgent.Agent({
```

Comment:
should we do this instead?
```suggestion
this.destHTTPAgent = new HttpsAgent.Agent(httpsConfig);
```

(requires that httpsAgent has been validated earlier)